### ISSUE TITLE
Misaligned calendar entriy highlighting on mobile

### ISSUE THREAD

**Issue by MrGreenTea** [YoungVision-eV/website#280]
![Screenshot_2024-07-16-09-49-16-913_org mozilla firefox](https://github.com/user-attachments/assets/1a7bf9d9-fc08-46ea-a524-83ed11147a57)

**Comment by Han2-Ro** [comment]
This is probably  happening because the text is to long. Maybe we should enforce a maximum text length in payload

**Comment by MrGreenTea** [comment]
You're absolutely right, didn't even notice that. The component is designed for three lines of text on mobile.

I guess a character limit would be good to have for the design not to break completely. I'm not entirely certain what the relationship between character count and line count is though, because the browser tries to break sentences if needed which can cause a single long word to get it's own whole line. But that should at worst be an edge case.

**Comment by MrGreenTea** [comment]
![](https://github.com/user-attachments/assets/3e3eb24e-728c-46c2-8089-675300c68c10)
